ui/pages/Report/components/SeqrStats.jsx

```diff
@@ -3,28 +3,43 @@ import { connect } from 'react-redux'
 import PropTypes from 'prop-types'
 import { Header, Table } from 'semantic-ui-react'
 
+import { getUser } from 'redux/selectors'
 import DataLoader from 'shared/components/DataLoader'
 import { DATASET_TITLE_LOOKUP } from 'shared/utils/constants'
 import { getSeqrStatsLoading, getSeqrStatsLoadingError, getSeqrStats } from '../selectors'
 import { loadSeqrStats } from '../reducers'
 
-const SeqrStats = React.memo(({ stats, error, loading, load }) => (
+const DEMO_COLUMN = { title: 'Demo Projects', key: 'demo' }
+const COLUMN_MAP = {
+  [true]: [
+    { title: 'Internal Projects', key: 'internal' },
+    { title: 'External AnVIL Projects', key: 'external' },
+    DEMO_COLUMN,
+    { title: 'No Access Projects', key: 'no_access' },
```

Comment:
Does the `No Access Projects` mean `inaccessible projects`? Consider renaming it.

Reply:
they aren't inaccessible because superusers and data managers do still have access, theres just no way to grant onyone else access. I think 'No AnVIL" will be clearer and will match the proposed API change